### PR TITLE
Always show launcher settings and some changes

### PR DESCRIPTION
**Pull request by kugiigi**
recreation of #235 for better commits

### CODE REVIEW

File: plugins/launcher/PageComponent.qml [hunk 1]
```diff
@@ -62,7 +65,7 @@ ItemPage {
             spacing: units.gu(1)
 
             SettingsItemTitle {
-                text: i18n.tr("On large screens:")
+                text: i18n.tr("On desktop mode:")
```
Comment (dobey) [inline plugins/launcher/PageComponent.qml:68]: Does the text here really need to be changed? If so, I think we can also choose better wording for this. But I don't think it needs to be changed.

**Review by dobey — CHANGES_REQUESTED** [review]

File: plugins/launcher/PageComponent.qml (hunk 1 above)
Comment (kugiigi) [reply plugins/launcher/PageComponent.qml:68]: It's not necessary to change but I feel like `On large screens` does not describe accurately when the settings take effect. I mean even in large screens, the launcher is always hidden when in staged mode. In desktop mode, the setting takes effect in certainty. Or maybe Windowed mode is a better term?

Comment (kugiigi) [reply plugins/launcher/PageComponent.qml:68]: I went back and re-read this topic and I think I'm going to stand with the words I used. Desktop mode is the official term used in Lomiri as displayed in the indicators. The option to always show the launcher only takes effect on desktop mode and not necessarily on large screens :smile:

Comment (dobey) [reply plugins/launcher/PageComponent.qml:68]: I think we should change the text in the indicator too then, not use "desktop" everywhere. Instead we should call it windowed mode.

Comment (kugiigi) [reply plugins/launcher/PageComponent.qml:68]: I would actually agree with that. Windowed mode describes it better then maybe in the future, tiling mode too :)

**Comment by UniversalSuperBox** [comment]
Hello,

Thank you for your contribution to system-settings. We have now moved this repository to GitLab to take advantage of better issue tracking and CI capabilities.

Things are going to be a little crazy for this repository between now and the release of Ubuntu Touch based on 20.04, but here's what you need to know:

* This PR was re-created automatically as a GitLab MR at https://gitlab.com/ubports/core/lomiri-system-settings/-/merge_requests/269. Unfortunately, if you would like to make changes to this MR on GitLab, you will need to fork the new repository, push your new changes, and create a new MR.
* You will probably want to work on the `ubports/xenial` branch when you are creating MRs. The `main` branch is only built for Ubuntu Touch based on 20.04, so you will be unable to test your changes for now.

If you have any questions, please comment on the MR that GitLab created for you. We'll try our best to answer.

**Pull request closed without merging**